PWGHF/TableProducer/derivedDataCreatorBplusToD0Pi.cxx

```diff
@@ -266,6 +268,10 @@ struct HfDerivedDataCreatorBplusToD0Pi {
         hfHelper.cosThetaStarBplus(candidate),
         ct);
     }
+    if (fillCandidateSel) {
+      rowCandidateSel(
+        BIT(candFlag));
+    }
```

Comment:
How is this useful?

Reply:
I might be mistaken but i thought it tracks where in the process something was or wasnt selected in (topol, PID, etc). If we just store signal candidates in the derived format that pass all the cuts perhaps its not useful but it could be useful if one stores bkg too. I put it so it would be uniform with the other deriveddatacreators (D0 and Lc) which have it.

Reply:
In the current code, the value of `candFlag` is either `0` or `1` to distinguish B+ and B-. But it can be changed to store the additional info about the selection steps.

Reply:
currently this is consistent with the D0 and Lc right?